frontend/src/components/Login.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button, TextField } from "@mui/material";
+import { useState } from "react";
+import { Navigate } from "react-router-dom";
+import { login } from "../services/APIService";
+import { isAuthenticated } from "../services/AuthService";
+
+function Login() {
+  const [username, setUsername] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+
+  const handleUsernameChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setUsername(e.target.value);
+  };
+
+  const handlePasswordChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setPassword(e.target.value);
+  };
+
+  if (isAuthenticated()) {
+    return <Navigate to="/passwords" replace />;
+  } else {
+    return (
+      <div
```

Comment:
https://mui.com/material-ui/react-tabs/
Use tabs to create a switcher between sign-up and login. Then we can save the hassle of using 2 components for it